index.html

```diff
@@ -15,229 +15,286 @@
+      <ul>
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+        <li>CapiBara</li>
+        <li>Selenium</li>
+      </ul>   
+      <img src="Images/ic_lamp.jpg" alt=""/>
+      
+    </div>
+  
+    <div class="cards third d-flex">
+      <header><h2>Skills</h2></header>
+      <ul>
+        <li>Codekit</li>
+        <li>CodePen</li>
+        <li>GitHub</li>
+        <li>GitLAb</li>
+        <li>Terminal</li>
+      </ul>    
+      <img src="Images/ic_skills.jpg" alt=""/>    
+    </div>
 
-  <div class="cards first d-flex">
-    <header><h2>Languages</h2></header>
-    <ul>
-      <li>Javascript (ES5/ES6)</li>
-      <li>Ruby</li>
-      <li>HTML</li>
-      <li>CSS</li>
-    </ul>   
-      <img src="Images/ic_asterisks.jpg" alt=""/>   
-  </div>  
+    <div class="cards desktop-card-fourth d-flex hide">
+      <img src="/Images/image_dots_with_teal.png" alt=""/> 
+    </div>
+  
+  </section>
+</div>
 
-  <div class="cards second d-flex">
-    <header><h2>Frameworks</h2></header>
-    <ul>
-      <li>Bootstrap</li>
-      <li>Ruby on Rails</li>
-      <li>CapiBara</li>
-      <li>Selenium</li>
-    </ul>   
-    <img src="Images/ic_lamp.jpg" alt=""/>
-    
+<div class="Desktop-Get-started">
+  <div class="contact-form-image">
+    <header class="header d-flex">
+      <img src="Images/geometry.png" alt="">            
+    </header>  
   </div>
-
-  <div class="cards third d-flex">
-    <header><h2>Skills</h2></header>
-    <ul>
-      <li>Codekit</li>
-      <li>CodePen</li>
-      <li>GitHub</li>
-      <li>GitLAb</li>
-      <li>Terminal</li>
-    </ul>    
-    <img src="Images/ic_skills.jpg" alt=""/>    
+  
+  <section class="content d-flex desktop-contact-form" id="contact-form"> 
+    <header class="header d-flex">
+      <h3 class="h1-style">Get started</h3>       
+    </header>  
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+      sed do eiusmod tempor incididunt ut labore et dolore magna.
+    </p> 
+    <form action="https://formspree.io/f/xwkawbvd" method="POST">
+      <fieldset class="contact-form-fieldset d-flex">
+        <div class="form-fieldset-labels">
+          <label class="fieldset-label d-flex" for="user_name">
+            Your name
+            <input type="text" id="user_name" required maxlength="30" placeholder="Anastasiia" name="user_name">
+          </label>
+          <label class="fieldset-label d-flex" for="user_email">
+            Your email
+            <input type="email" id="user_email" required placeholder="example@email.com" name="user_email">
+          </label>
+        </div>    
+        <label class="fieldset-label d-flex" for="user_message">Message</label>   
+        <textarea class="textarea-text" required id="user_message" name="user_message" maxlength="500" placeholder="Enter your message here..."></textarea>
+        <button class="collaboration-button" type="submit"><span class="collaboration-button-span">Start collaboration</span></button>
```

Comment:
- [ ] Kindly let's reduce the height of this button (Start collaboration), it seems to be bigger than the one in Figma, You can give it something like 6% of height instead of 10%.
- [ ] You can do the same for the (Get my CV) button as well 😉